packages/cli-platform-android/src/commands/buildAndroid/startServerInNewWindow.ts

```diff
@@ -17,19 +18,19 @@ export function startServerInNewWindow(
     : 'launchPackager.command';
   const packagerEnvFilename = isWindows ? '.packager.bat' : '.packager.env';
   const portExportContent = isWindows
-    ? `set RCT_METRO_PORT=${port}`
-    : `export RCT_METRO_PORT=${port}`;
+    ? `set RCT_METRO_PORT=${port}\nset PROJECT_ROOT=${projectRoot}`
+    : `export RCT_METRO_PORT=${port}\nexport PROJECT_ROOT=${projectRoot}`;
```

Comment:
I'm wondering if we still need those files. 
You have moved it out of the react-native context so I believe it's no longer used by any of the building scripts.
Maybe instead of doing it this strange way, we can pass those environment variables as an argument.

Reply:
Due to `open` limitations we need this additional step.
Feel free to resolve this.